docs-gen/Makefile

```diff
@@ -1,5 +1,11 @@
 build: clean
 	hugo --minify
+	@# Workaround for https://github.com/google/docsy/issues/350 and https://github.com/gohugoio/hugo/issues/7714
+	$(eval URL_PATH = $(shell grep baseURL config.toml | sed "s/'//g" |  cut -d '/' -f 4))
+	@ if [ ! -z "$(URL_PATH)" ]; \
+	  then \
+	    sed -i.bak "s/background-image:url(/background-image:url(\/$(URL_PATH)/g" ../docs/index.html && rm ../docs/index.html.bak; \
```

Comment:
It looks like `-i ''` won't create a backup file https://thoughtbot.com/blog/sed-102-replace-in-place

Reply:
Apparently we can't because OSX wants `-i ''` and Linux `-i`.